airflow/operators/sensors.py

```diff
@@ -52,10 +52,10 @@ def poke(self, context):
         raise AirflowException('Override me.')
 
     def execute(self, context):
-        started_at = datetime.now()
+        self.started_at = datetime.now()
```

Comment:
This is more of a shim for testing. Ideally, I would want to have a test case that prevents a regression there. Instead of modifying the timeout duration, I can then modify started_at at the instance level to make it further in the past and not have to wait over a day for testing to finish.